src/Shared/WebEncoders/WebEncoders.cs

```diff
@@ -343,6 +344,7 @@ public static int GetArraySizeRequiredToEncode(int count)
         /// </summary>
         /// <param name="input">The binary input to encode.</param>
         /// <returns>The base64url-encoded form of <paramref name="input"/>.</returns>
+        [SkipLocalsInit]
```

Comment:
GH doesn't let me comment on L359, but this should be
```c#
? stackalloc char[128]
```
to use the full power of `SkipLocalsInit`.

Note: the code is safe, as the result is sliced later on, so it doesn't matter if any larger buffer is passed to `Base64UrlEncode`.